core/spec/models/spree/order/checkout_spec.rb

```diff
@@ -646,11 +646,11 @@ class SubclassedOrder < Spree::Order
           [payments_attributes: Spree::PermittedAttributes.payment_attributes]
       end
 
-      let(:credit_card) { create(:credit_card, user_id: order.user_id) }
+      let(:credit_card) { create(:credit_card, user: order.user) }
 
       let(:params) do
         ActionController::Parameters.new(
-          order: { payments_attributes: [{payment_method_id: 1}], existing_card: credit_card.id },
+          order: { payments_attributes: [{payment_method_id: 1}], existing_user_payment_source_id: credit_card.user_payment_source.id },
```

Comment:
Line is too long. [136/120]<br>Space inside { missing.<br>Space inside } missing.